dev-util/conan/conan-1.19.2.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6} )
```

Comment:
Any chance you could try with 3.7? Often just adding it, then running repoman, is good indicator if its supported or not. Then you have python_test enabled now, so its easy to test whether it works or not, if deps are satisfied.

Reply:
dev-python/patch don't have python3_7, so no luck